tests/bluetooth/df/connectionless_cte_chains/src/test_add_cte_to_chain.c

```diff
@@ -121,6 +124,47 @@ void test_add_cte_for_not_all_pdu_in_chain(void)
 	/* Teardown */
 	common_release_per_adv_chain(adv);
 	common_release_adv_set(adv);
+	lll_adv_init();
+}
+
+void test_add_cte_to_not_all_pdus_in_chain_enqueued_to_lll(void)
+{
+	struct pdu_adv *pdu_prev, *pdu_new;
+	struct ll_adv_set *adv;
+	uint8_t handle;
+	uint8_t upd;
+	int err;
+
+	/* Setup for test */
+	adv = common_create_adv_set(TEST_ADV_SET_HANDLE);
+	/* Use the same number for PDUs in a chain as for CTE request */
+	common_prepare_df_cfg(adv, TEST_CTE_COUNT);
+	common_create_per_adv_chain(adv, TEST_PER_ADV_CHAIN_LENGTH);
+	common_validate_per_adv_chain(adv, TEST_PER_ADV_CHAIN_LENGTH);
+
+	/* Swap PDU double buffer and get new latest PDU data */
+	pdu_new = lll_adv_sync_data_latest_get(adv->lll.sync, NULL, &upd);
+	zassert_not_equal(pdu_new, NULL,
+			  "Unexpected value of PDU pointer after PDU double buffer swap");
+
+	pdu_prev = lll_adv_sync_data_peek(adv->lll.sync, NULL);
+	zassert_equal(pdu_prev, pdu_new,
+		      "Unexpected value of PDU pointer after PDU double buffer swap");
```

Comment:
I think debugging any issues would be easier if these two didn't have the exact same assert message.

Reply:
Yes, good point. Corrected.